frontend/src/hooks/api/actions/useApi/useApi.ts

```diff
@@ -108,6 +108,10 @@ const useAPI = ({
                 const response = await res.json();
                 if (response?.details?.length > 0 && propagateErrors) {
                     const error = response.details[0];
+                    setErrors(prev => ({
```

Comment:
this was missign before. When unknown error was triggered the errors Record was empty. Code that depended on errors being populated (last import stage) failed because of this check missing.